releasenotes/notes/worker-utilization-in-telemetry-15708e06cd138ea7.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Added `workers.utilization` to the telemetry. This metric represents the amount of time that a runner worker has been running checks.
```

RST uses double backticks.

```suggestion
    Added ``workers.utilization`` to the telemetry. This metric represents the amount of time that a runner worker has been running checks.
```